server-side-rendering/server/lib/appwrite.ts

```diff
@@ -0,0 +1,37 @@
+import { Client, Account } from "node-appwrite";
+
+export const SESSION_COOKIE = "my-custom-session";
+
+type Event = Parameters<Parameters<typeof defineEventHandler>[0]>[0];
+
+export function createAdminClient() {
+  const client = new Client()
+    .setEndpoint(process.env.APPWRITE_ENDPOINT!)
```

Comment:
These environment variables do not exist, I think you use `APPWRITE_PUBLIC_....`